scripts/train_parser_and_tagger.py

```diff
@@ -95,18 +98,28 @@ def train_parser_and_tagger(train_json_path: str,
     else:
         optimizer = nlp.begin_training(lambda: train_corpus.train_tuples)
     nlp._optimizer = None
-    print("Itn.  Dep Loss  NER Loss  UAS     NER P.  NER R.  NER F.  Tag %   Token %  CPU WPS  GPU WPS")
-    train_docs = train_corpus.train_docs(nlp, projectivize=True)
+
```

Comment:
Did you check that this is set to "true" internally somehow? (I'm pretty sure all the trees are projective, but still)

Reply:
yup, its here https://github.com/explosion/spaCy/blob/a3efa3e8d9ca4c54ad61fa7352a485b79acec04b/spacy/gold.pyx#L192

Reply:
Huh now the constructor argument does nothing, weird